Documentation/ceph-cluster-crd.md

```diff
@@ -672,6 +672,18 @@ spec:
 
 ### External cluster
 
+**The minimum supported Ceph version for the External Cluster is Luminous 12.2.x.**
+
+The features available from the external cluster will vary depending on the version of Ceph. The following table shows the minimum version of Ceph for some of the features:
+
+| FEATURE                                      | CEPH VERSION |
+|----------------------------------------------|--------------|
+| Dynamic provisioning RBD                     | 12.2.X       |
+| Configure extra CRDs (object, file, nfs)[^1] | 13.2.3       |
```

Comment:
Is the reference supposed to be rendered as a link? Looks like it's just text.

Reply:
Typically in markdown, it's an annotation for a footnote.